dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -108,14 +108,24 @@ else
 fi
 
 CHE_IMAGE_REPO=${CHE_IMAGE_REPO:-${DEFAULT_CHE_IMAGE_REPO}}
-DEFAULT_CHE_IMAGE_TAG="spi"
+DEFAULT_CHE_IMAGE_TAG="nightly"
```

Comment:
nightly is coming from `che6` branch ?

Reply:
we don't have spi tags any more, it should be nightly as in master. and nightly is default tag

Reply:
default should be for now `che6` ?
https://hub.docker.com/r/eclipse/che-server/tags/

Reply:
yes but AFAIK for now che6 scripts should use che6 binaries by default

Reply:
we can't change that for each branch, and to not forget change it before merge to master I would prefer to keep at as in master and use ENV var to override. Because I'm 100% it will be merged in master with che6 tags

Reply:
we don't change that for each branch, only "master" and "che6"

User running scripts with CLI 6.0.0-M1 will expect that it's running the correct "branch"/image by default

Reply:
It would be more convenient for development process to have here the same default value as docker images build scripts have. And it is `nightly`.

Reply:
yeah but I think there is something wrong because local and `dockerhub` values should match the same targeted branch.
you use the cli to install the scripts in a folder, so you expect the cli install the scripts for the cli version you've used.

as it's a puppet module, couldn't we have a property value that is automatically setting the correct value based on the image that we're building ?

Reply:
We agreed to set che6 tag